src/services/lease-service/get-tenant.ts

```diff
@@ -137,6 +137,10 @@ async function fetchTenant(params: {
     data: {
       ...contact.data,
       address: contact.data.address,
+      isAboutToLeave:
+        !upcomingHousingContract &&
+        !!currentHousingContract &&
```

Comment:
```suggestion
        currentHousingContract &&
```

should be truthy without the coercion, right?

Reply:
Ah makes sense!